test/test_codec.py

```diff
@@ -205,3 +205,19 @@ def test_import_export_additional_frame_info(self):
             with open(out_file_name, "r") as file:
                 data = file.read()
             self.assertIn("UserFrameAttr", data)
+
+    def test_export_jsonall(self):
+        """Check the jsonAll doesn't raise and export some additional field."""
+        db_path = os.path.join(
+            os.path.dirname(__file__), "..", "test", "test.dbc")
```

Comment:
Is it plausible to make a minimal example 'file' here (`io.BytesIO`) that covers just this case?  Or maybe it doesn't even need to be a file and could just be built up in a few lines from raw canmatrix objects.

Reply:
Built up from raw canmatrix objects.